scripts/release/mule/sign/sign.sh

```diff
@@ -16,75 +15,47 @@ BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}
 CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
 PKG_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
 SIGNING_KEY_ADDR=dev@algorand.com
+STATUSFILE="build_status_${CHANNEL}_${VERSION}"
+
+cd "$PKG_DIR"
 
 if ! $USE_CACHE
 then
-    export ARCH_BIT
-    export ARCH_TYPE
-    export CHANNEL
-    export OS_TYPE
-    export VERSION
-
-    if [ "$PKG_TYPE" == "tar.gz" ]
-    then
-        mule -f package-sign.yaml package-sign-setup-tarball
-    else
-        mule -f package-sign.yaml "package-sign-setup-$PKG_TYPE"
-    fi
-fi
-
-make_hashes () {
-    # We need to futz a bit with "source" to make the hashes correct.
-    local HASH_TYPE=${1:-$PKG_TYPE}
-    local PACKAGE_TYPE=${2:-$PKG_TYPE}
-
-    HASHFILE="hashes_${CHANNEL}_${OS_TYPE}_${ARCH_TYPE}_${VERSION}_${HASH_TYPE}"
-    # Remove any previously-generated hashes.
-    rm -f "$HASHFILE"*
+    # deb
+    aws s3 cp "s3://algorand-staging/releases/$CHANNEL/$VERSION/algorand_${CHANNEL}_${OS_TYPE}-${ARCH_TYPE}_${VERSION}.deb" .
+    aws s3 cp "s3://algorand-staging/releases/$CHANNEL/$VERSION/algorand-devtools_${CHANNEL}_${OS_TYPE}-${ARCH_TYPE}_${VERSION}.deb" .
 
-    {
-        md5sum *"$VERSION"*."$PACKAGE_TYPE" ;
-        shasum -a 256 *"$VERSION"*."$PACKAGE_TYPE" ;
-        shasum -a 512 *"$VERSION"*."$PACKAGE_TYPE" ;
-    } >> "$HASHFILE"
+    # rpm
+    aws s3 cp "s3://algorand-staging/releases/$CHANNEL/$VERSION/algorand-${VERSION}-1.${ARCH_BIT}.rpm" .
+    aws s3 cp "s3://algorand-staging/releases/$CHANNEL/$VERSION/algorand-devtools-${VERSION}-1.${ARCH_BIT}.rpm" .
+fi
 
-    gpg -u "$SIGNING_KEY_ADDR" --detach-sign "$HASHFILE"
-    gpg -u "$SIGNING_KEY_ADDR" --clearsign "$HASHFILE"
-}
+# TODO: "$PKG_TYPE" == "source"
 
-make_sigs () {
-    local PACKAGE_TYPE=${1:-$PKG_TYPE}
+# Clean package directory of any previous operations.
+rm -rf hashes* *.sig *.asc *.asc.gz
 
-    # Remove any previously-generated signatures.
-    rm -f ./*"$VERSION"*."$PACKAGE_TYPE".sig
+for file in *.tar.gz *.deb
+do
+    gpg -u "$SIGNING_KEY_ADDR" --detach-sign "$file"
+done
 
-    for item in *"$VERSION"*."$1"
-    do
-        gpg -u "$SIGNING_KEY_ADDR" --detach-sign "$item"
-    done
-}
+for file in *.rpm
+do
+    gpg -u rpm@algorand.com --detach-sign "$file"
```

Comment:
I find it weird that we parameterize `$SIGNING_KEY_ADDR` but then hardcode `rpm@algorand.com`. Maybe do both, and we can do "test" signing, or sign off on nightlies with a different key?

Reply:
The reason is because the only the rpm packages use this user to sign and everything else uses the dev@ user.  We could stick that in a variable, but it's only used once...

Yes, we'll need to sign nightlies with a different key, but I'm not sure how that will affect these scripts yet.

Reply:
We don't allow overriding `$SIGNING_KEY_ADDR` right now, so I'll grudgingly approve it, but this is probably better variabled out, because I can almost guarantee we'll want to generalize it for nightly / bespoke builds.

Reply:
Yes, I agree, I'd just rather not do that before we actually tackle that.